### ISSUE TITLE
Bug: Reports - Last Visit in some cases is not being updated.

### ISSUE THREAD

**Issue by kimwhite** [strangerstudios/paid-memberships-pro#2066]
**Describe the bug**
The "Last Visit" cell is either N/A or unchanged when members login and log out.
This keeps Visits This Week and Visits This Month from populating also.

**To Reproduce**
1. Go to Reports Tab
2. Go to "Visits, Views, and Logins" click the "Details" button
3. See dates on Last Visit and Last Login columns.
4. Test user has had little activity by logging in, surfing, log out.
5.  Last Login will update but "Last Visit" date does not

**Screenshots**
![Screen Shot 2022-05-06 at 11 27 38 AM](https://user-images.githubusercontent.com/8083630/167164317-4b0416f2-dec6-419c-b73d-2fd0b2a65780.png)

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only Paid Memberships Pro plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Memberlite](https://wordpress.org/themes/memberlite/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
WordPress 5.9.3
PMPro 2.8

</details>

**Comment by dwanjuki** [comment]
Another customer ran into this issue. Mods #567242
I was able to replicate on PMPro 3.0.4

**Comment by dwanjuki** [comment]
Another customer ran into this issue. Mods: 574920

**Comment by MaryOJob** [comment]
Another issue on this here, please: [#631462] (Moderators Only)
On my live testing site, this is similar but I have cookies blocked on my browser

**Comment by ipokkel** [comment]
While testing this issue in a local testing environment with PMPro 3.4.3 I could replicate this issue.

![Image](https://github.com/user-attachments/assets/e0046d11-b6fc-4447-ac83-0586c850b717)

**Comment by dwanjuki** [comment]
Another customer noted this issue.
Mods: 693653

**Comment by dwanjuki** [comment]
Another customer noted this issue.
Mods: 705942

Replicated on PMPro 3.7.4
